helm-v3/005-plugins.md

```diff
@@ -63,8 +68,28 @@ used for executing Lua plugins mirrors the pattern for writing command-line
 applications. A `main()` function will be executed with access to STDIN, STDOUT,
 STDERR, and all command-line parameters and options.
 
-### Lua Plugin Sandboxing
+#### Lua Plugin Sandboxing
 
 Like charts, each plugin will be executed in an isolated Lua sandbox. Unlike
 chart scripts, plugins will be granted access to all core libraries, and will
 not require security prompts to accept.
+
+## Installing Plugins
+A single plugin can be installed via `helm plugin install`.
+
+A set of plugins can also be defined in a yaml file and installed during the `helm init` process by using a flag called `--custom plugins <file.yaml>`
```

Comment:
Shouldn't it be `--custom-plugins <file.yaml>` (notice the - between custom and plugins)?

Reply:
You're right! Instead of fixing that though, I dropped the `custom` and named that flag `--plugins` instead. Let me know what you think.